### ISSUE TITLE
UAT: Size Selection Missing Tooltips

### ISSUE THREAD

**Issue by awilcoxa** [magento-research/pwa-studio#701]
On the Product Display Page, when a product has size configuration and a size is selected the size swatch does not correctly display a tooltip with the full name of the size. Sizes are also displayed in reverse order (largest to smallest), sizes should be displayed from smallest to largest. 

Issue 20: https://wiki.corp.magento.com/display/FF/Venia+UX+Issues 

## This issue is for the following packages:

- [x ] `venia-concept`
- [ ] `pwa-buildpack`
- [x ] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

## This issue is a:

- [x ] Bug
- [ ] Feature suggestion
- [ ] Documentation issue
- [ ] Other (Please Specify)

**Comment by awilcoxa** [comment]
Current: 

![image](https://user-images.githubusercontent.com/44207440/50299757-0df7a280-0448-11e9-8628-2d365a99d17a.png)

**Comment by AlexeyKaryka** [comment]
Having tooltips for size swatches is also hard-reaching because data with configurable options for sizes currently doesn't contain information about full size names, I assume that creating mapping with short size name - full size name pairs isn't quite a good solution. So it's need to fill information about full size names into configurable_options of sizes apart from short size names

**Comment by Starotitorov** [comment]
@zetlen, @jimbo, @supernova-at, I guess we need discuss the question related to sorting size options and `value_index` property. Should we just display size options sorted by `value_index` in descending order or it is better to change `value_index` for each size option in Magento CMS?

**Comment by Starotitorov** [comment]
@zetlen, @jimbo, @supernova-at, I agree with this comment https://github.com/magento-research/pwa-studio/issues/701#issuecomment-449570347, it is better to add a separate property with full size name to each size option and receive this information from the backend. What do you think, guys?
